src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -133,7 +133,65 @@ const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log)
       '"logging.timezone" has been deprecated and will be removed ' +
         'in 8.0. To set the timezone moving forward, please add a timezone date modifier to the log pattern ' +
         'in your logging configuration. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.md'
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx'
+    );
+  }
+  return settings;
+};
+
+const destLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.dest')) {
+    log(
+      '"logging.dest" has been deprecated and will be removed ' +
+        'in 8.0. To set the destination moving forward, you can use the "console" appender ' +
+        'in your logging configuration or define a custom one. For more details, see ' +
+        'https://github/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const quietSilentVerboseLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (
+    has(settings, 'logging.quiet') ||
+    has(settings, 'logging.silent') ||
+    has(settings, 'logging.verbose')
+  ) {
+    log(
+      '"logging.quiet", "logging.silent" and "logging.verbose" have been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level" in your logging configuration. ' +
+        'Levels "error", "off" and "all" are equivalent to "quiet", "silent" and "verbose". For more details, see ' +
+        'https://github/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const jsonLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.json')) {
+    log(
+      '"logging.json" has been deprecated and will be removed ' +
+        'in 8.0. To specify log message format moving forward, ' +
+        'you can configure the "appender.layout" property for every custom appender in your logging configuration. There is currently no default layout for custom appenders and each one must be declared explicitly. For more details, see ' +
+        'https://github/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const logRotateDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
```

Comment:
At the moment, `logging.rotate` sub-options are all flagged as `[experimental]`. Strictly speaking, we shouldn't need to mention these are deprecated in 8.0, however, I added them because we do have the `rolling-file` appender in the KP logging system.